_config.yml

```diff
@@ -78,6 +68,24 @@ collections:
   promos:
     filename: "{{ published_at | date: '%Y-%m-%d-%H:%M:%S' }}-{{ title | slugify }}-{{ slug }}"
     output: false
+  articles:
+    output: false
+  topics:
+    output: false
+  authors:
+    output: false
+  podcasts:
+    output: false
+  episodes:
+    output: false
+  tags:
+    output: false
+  features:
+    output: false
+  perspectives:
+    output: false
+  categories:
+    output: false
```

Comment:
@adamtgeorge You can explicitly link to `/topics` by adding the following line below this line:

```yml
  permalink: /topics/:title
```

[See here](https://github.com/crdschurch/crds-media/blob/development/_config.yml#L45) for how it is on media.